packages/editor/src/store/actions.js

```diff
@@ -813,7 +817,8 @@ export function* resetEditorBlocks( blocks, options = {} ) {
 	const edits = { blocks: yield* getBlocksWithSourcedAttributes( blocks ) };
 
 	if ( options.__unstableShouldCreateUndoLevel !== false ) {
-		edits.content = serializeBlocks( edits.blocks );
+		edits.content = ( { blocks: blocksForSerialization = [] } ) =>
+			serializeBlocks( blocksForSerialization );
```

Comment:
So this is called on each change right? What's the reasoning? potentially it could be just called once right? I think calling it here is probably better but curious about the reasoning.

Reply:
To create a new undo level/edit. It's a new function reference so it's a new edit.

Reply:
ok makes sense.